[i 0802 07:24:34.227406 48 compiler.py:956] Jittor(1.3.8.5) src: /mnt/fast/nobackup/scratch4weeks/ae01116/zegenv/lib/python3.9/site-packages/jittor
[i 0802 07:24:34.231159 48 compiler.py:957] g++ at /usr/bin/g++(7.5.0)
[i 0802 07:24:34.231225 48 compiler.py:958] cache_path: /user/HS400/ae01116/.cache/jittor/jt1.3.8/g++7.5.0/py3.9.19/Linux-4.18.0-4x1a/AMDEPYC753232-x73/default
[i 0802 07:24:34.236130 48 __init__.py:411] Found nvcc(11.0.221) at /usr/local/cuda/bin/nvcc.
[i 0802 07:24:34.240679 48 __init__.py:411] Found addr2line(2.30) at /usr/bin/addr2line.
[i 0802 07:24:34.383022 48 compiler.py:1011] cuda key:cu11.0.221_sm_86
[i 0802 07:24:34.981719 48 __init__.py:227] Total mem: 503.74GB, using 16 procs for compiling.
[i 0802 07:24:39.125002 48 jit_compiler.cc:28] Load cc_path: /usr/bin/g++
[i 0802 07:24:39.326230 48 init.cc:62] Found cuda archs: [86,]
[w 0802 07:24:39.462048 48 compiler.py:1384] CUDA arch(86)>80 will be backward-compatible
[i 0802 07:24:41.437210 48 __init__.py:411] Found mpicc(2.1.1) at /usr/bin/mpicc.
[i 0802 07:24:51.257051 48 cuda_flags.cc:49] CUDA enabled.
/mnt/fast/nobackup/users/ae01116/multi-modal-dissertation-uos/jseg/models/losses/cross_entropy_loss.py:55: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with Jittor official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
[e 0802 07:24:55.790997 48 log.cc:258] Caught SIGINT, quick exit
